modules/dnsbl_tables.c

```diff
@@ -206,7 +206,12 @@ static ci_vector_t  *resolv_hostname(char *hostname)
 
     if (vect) {
         for (cur = res; cur != NULL; cur = cur->ai_next) {
-            memcpy(&(addr.sockaddr), cur->ai_addr, CI_SOCKADDR_SIZE);
+#ifndef USE_IPV6
```

Comment:
The modules should not care if support for IPv6 is enabled or not. The ci_sockaddr_t_to_ip which is called later,  should just return NULL if the given IP address is an IPv6 address and IPv6 is not supported. However the ci_sockaddr_t_to_ip is buggy so this is required for now.